Source/Core/Common/Hash.h

```diff
@@ -10,14 +10,16 @@
 
 namespace Common
 {
-u32 HashFletcher(const u8* data_u8, size_t length);  // FAST. Length & 1 == 0.
-u32 HashAdler32(const u8* data, size_t len);         // Fairly accurate, slightly slower
-u32 HashEctor(const u8* ptr, size_t length);         // JUNK. DO NOT USE FOR NEW THINGS
+u32 HashFletcher(const u8* data, size_t len);
```

Comment:
The `Length & 1 == 0` comment seems kind of important?

Reply:
I was actually hoping to replace it with some other hash function since this is only used in one place but wasn’t sure if that would break things. I’ll probably change it to take a type so then it can static assert about its size requirements.

Reply:
I moved it into NativeVertexFormat.h